BenchmarkCNV/BenchmarkCNV.inputs.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Benchmark_CNV_Caller.wittyer_docker": "yg96/wittyer:v2",
+  "Benchmark_CNV_Caller.gatk_docker": "us.gcr.io/broad-gatk/gatk:4.3.0.0",
+  "Benchmark_CNV_Caller.wittyer_sv_config": "gs://gptag/tag_1455_Dragen_CNV_Caller_Validation/wittyerConfigFile/sv-config.json",
+  "Benchmark_CNV_Caller.wittyer_cnv_evaluation_mode": "CrossTypeAndSimpleCounting",
+  "Benchmark_CNV_Caller.wittyer_sv_evaluation_mode": "SimpleCounting",
+  "Benchmark_CNV_Caller.wittyer_cnv_config": "gs://gptag/tag_1455_Dragen_CNV_Caller_Validation/wittyerConfigFile/cnv-config.json",
+  "Benchmark_CNV_Caller.variant_callset": "gs://gptag/tag_1455_Dragen_CNV_Caller_Validation/1KGP_3202.gatksv_svtools_novelins.freeze_V3_fixed.wAF.vcf",
+  "Benchmark_CNV_Caller.truth_sample_name": "HG00513",
+  "Benchmark_CNV_Caller.query_sample_name": "SM-GZQKA",
+  "Benchmark_CNV_Caller.eval_sv_vcf": "gs://fc-55e318f5-3e28-42f5-8d93-5e49aaabcb03/hg38_ml/HG00513_HG00513_1_SM-GZQKA_v1/HG00513_HG00513_1_SM-GZQKA_v1.sv.vcf.gz",
```

Comment:
I think it's difficult to keep track of files in these firecloud buckets.  Would it maybe be better to put it in gs://gptag/tag_1455_Dragen_CNV_Caller_Validation/

Reply:
I refer to the file paths to a terra workspace google bucket which Edyta created. https://app.terra.bio/#workspaces/broadtagteam/DRAGEN_v_3_10_4_CNV_validation 

Do we want to upload the same file to the gptag bucket?

Reply:
This comment need to be address together with @edytamalolepsza